.github/workflows/verify_failure.yml

```
@@ -0,0 +1,35 @@
+name: Verify Assertion Failure
+
+on:
+  push:
+    branches: 
```

actionlint (1.7.8)

string should not be empty

(syntax-check)